src/test/java/ScattTest.java

```diff
@@ -25,10 +25,11 @@ public void testHello() {
     @Test
     public void testGui() {
```

Comment:
At the time it was my best way to check that the UI was working. We need to design a better test for this. I think last clint and I talked we decided that we need to mock a UI. But overall with the way we implemented the the GUI I am still not sure the best way to test this.